### ISSUE TITLE
Initial location robot changed to 0,0 after reset

### ISSUE THREAD

**Issue by knmcguire** [cyberbotics/webots#6467]
**Describe the Bug**
I noticed with the crazyflie_apartment example, that if I reset the simulation, the bot goes from it's initial location (-5, -2) in the .wbt, to (0,0). This way the bot is stuck in the wall.

**Steps to Reproduce**
1. Start up the example world bitcraze/crazyflie_example
2. Play the simulator for a few seconds (crazyflie takes off but that is not the problem)
3. Reset the simulator with ![image](https://github.com/cyberbotics/webots/assets/11061260/796ed4ca-ea5d-4ee0-986f-abf69af7a808)
4. The crazyflies moves away from view and to (0,0), and now stuck in the wall
5. Reload the world with ![image](https://github.com/cyberbotics/webots/assets/11061260/7c1205fe-a899-4580-a11e-669f10438ec2) to get it back in position

**Expected behavior**
I expect the bot to return to its initial location of (-5, -2)

**System**
 - Operating System: Ubuntu 22.04
 - Graphics Card: Nvidia Geforce 3060 Mobile

**Comment by ShuffleWire** [comment]
In the Crazyfly Robot PROTO, on line 32, there is
`    translation 0 0 0.015`
Which afaik is causing the issue, and should be removed.
I'm correct ?

**Comment by knmcguire** [comment]
Yes absolutely you are correct. I'll remote that immediately from the proto now with an PR. Thanks!
